engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -222,6 +223,7 @@ public Result execute( Params params ) throws Throwable {
         return exitWithStatus( CommandExecutorCodes.Kitchen.COULD_NOT_LOAD_JOB.getCode() ); // same as the other list options
       }
 
+      MDC.put( KettleConstants.UUID, params.getUuid() );
```

Comment:
These MDC commands here seem a bit out of context.
Not a blocker, but would be nice to have a comment to provide context

Reply:
Sure, there's a merge conflict on an import, I can add a comment when I fix that.